src/plugin-store.svelte

```diff
@@ -205,6 +207,7 @@ let pluginStore: Element;
     :root,
     :host {
         --header-height: 128px;
+        scrollbar-color: var(--color-grey-2) var(--mdc-theme-surface);
```

Comment:
Changed scrollbar background color to be consistent with off-white tinted background.